README.md

```diff
@@ -1,101 +1,288 @@
+
+``` ruby
+include Phaxio::Resources
 
-To send a fax:
+fax_file = File.open 'test.pdf', 'rb'
+Fax.create to: '+15558675309', file: fax_file
+```
 
-    Phaxio.send_fax(to: "0123456789", filename: File.new("test.pdf"))
 
 ### Currently Supported API Calls
 
-* send_fax - `Phaxio.send_fax(to: "0123456789", filename: File.new("test.pdf"))`
-* resend_fax - `Phaxio.resend_fax(id: 1234)`
-* test_receive - `Phaxio.test_receive(filename: "test_file.pdf")`
-* provision_number - `Phaxio.provision_number(area_code: 802)`
-* release_number - `Phaxio.release_number(number: "8021112222")`
-* list_numbers - `Phaxio.list_numbers(area_code: 802)`
-* get_fax_file - `Phaxio.get_fax_file(id: 123456, type: p)`
-* list_faxes - `Phaxio.list_numbers(area_code: 802)`
-* list_faxes - `Phaxio.list_faxes(start: Time.now - 48000,
-end: Time.now)`
-* get_fax_status - `Phaxio.get_fax_status(id: 123456)`
-* cancel_fax - `Phaxio.cancel_fax(id: 123456)`
-* delete_fax - `Phaxio.delete_fax(id: 1234, files_only: true)`
-* get_account_status - `Phaxio.get_account_status`
-
-### Example
-
-    require 'phaxio'
-
-    Phaxio.config do |config|
-      config.api_key = "your_key"
-      config.api_secret = "your_secret"
-    end
+#### Faxes
 
-    @fax = Phaxio.send_fax(to: '15555555555', string_data: "hello world")
-    Phaxio.get_fax_status(id: @fax["faxId"])
+##### `Fax.create`
 
-    # Get a Fax and save it as a PDF
-    @pdf = Phaxio.get_fax_file(id: @fax["faxId"], type: "p")
-    File.open("received_test.pdf", "w") do |file|
-      file << @pdf
-    end
+Create and send a fax.
+
+``` ruby
+fax_file = File.open 'test.pdf', 'rb'
+ref = Fax.create to: '+15558675309', file: fax_file
+# => Fax::Reference(id: 1234)
+fax = ref.get
+# => Fax(id: 1234, num_pages: 1, ...)
+```
+
+##### `Fax.list`
+
+List faxes in date range.
+
+``` ruby
+start = 2.weeks.ago
+stop = 1.week.ago
+faxes = Fax.list created_after: start, created_before: stop
+# => Phaxio::Resource::Collection([Fax(id: 1234, ...), ...])
+faxes.length
+# => 5
+faxes.map(&:cost).inject(&:+)
+# => 35
+```
+
+##### `Fax.get`
+
+Get information about a specific fax.
+
+``` ruby
+fax = Fax.get 1234
+# => Fax(id: 1234, ...)
+```
+
+##### `Fax.cancel`
+
+Cancel a fax.
+
+``` ruby
+Fax.cancel 1234
+# => Fax::Reference(id: 1234)
+```
+
+##### `Fax.resend`
+
+Resend a fax.
+
+``` ruby
+Fax.resend 1234
+# => Fax::Reference(id: 5678)
+```
+
+##### `Fax.delete`
+
+Delete a fax. Only test faxes are allowed to be deleted.
+
+``` ruby
+Fax.delete 1234
+# => true
+```
+
+##### `Fax.delete_file`
+
+Delete fax file.
+
+``` ruby
+Fax.delete_file 1234
+# => true
+```
+
+##### `Fax.file`
+
+``` ruby
+Fax.file 1234
+# => File
+```
+
+##### `Fax.test_receive`
+
+Test receiving a fax.
+
+``` ruby
+fax_file = File.open 'test.pdf', 'rb'
+Fax.test_receive file: fax_file
+# => true
+```
+
+##### `Fax.supported_countries`
+
+Get a list of supported countries.
+
+``` ruby
+Fax.supported_countries
```

Comment:
In the other v2 client libs, we do something like `Public::Country.list` to more closely match the API path.